composeApp/src/androidInstrumentedTest/kotlin/org/ooni/probe/uitesting/screenshots/AutomateScreenshotsTest.kt

```diff
@@ -234,4 +245,233 @@ class AutomateScreenshotsTest {
                 Screengrab.screenshot("16-about")
             }
         }
+
+    @Test
+    fun results() =
+        runTest {
+            skipOnboarding()
+            setupTestResults()
+            start()
+            with(compose) {
+                wait { onNodeWithContentDescription(Res.string.app_name).isDisplayed() }
+
+                clickOnText(Res.string.TestResults_Overview_Title)
+
+                wait { onNodeWithText("Websites").isDisplayed() }
+
+                Screengrab.screenshot("17-results")
+
+                clickOnText("Websites")
+
+                wait(10.seconds) { onNodeWithText("https://z-lib.org/").isDisplayed() }
```

Comment:
We can use a composite statement here and check if multiple UI elements are displayed. This would help avoid the need to do `Thread.sleep()`

Reply:
let me run again and check.